docs/getting-started/artifact-control.asciidoc

```diff
@@ -6,8 +6,18 @@
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [secure, manage]
 
-[sidebar]
---
-[.text-center]
-This page is a placeholder for future documentation.
---
+On the **Protection updates** tab of the {elastic-defend} integration policy, you can configure how {elastic-defend} receives updates from Elastic with the latest threat detections, global exceptions, malware models, rule packages, and other protection artifacts. By default, these artifacts are updated automatically every day, ensuring your environment is up to date with the latest protections. 
```

Comment:
> By default, these artifacts are updated automatically every day
We should probably just say "regularly" since we don't necessarily have daily updates